airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -240,10 +240,10 @@ def _get_init_action_timeout(self):
         match = re.match(r"^(\d+)([sm])$", self.init_action_timeout)
         if match:
             if match.group(2) == "s":
-                return self.init_action_timeout
+                return {"seconds": self.init_action_timeout}
```

Comment:
It seems value under `seconds` key will be of `str` type in this case, and an `int` type in the other - is that intended/valid?

Reply:
Fixed